server/src/serde_types.in.rs

```diff
@@ -0,0 +1,68 @@
+#[derive(Serialize, Deserialize)]
+struct Error {
+    code: i32, //unique code identifying error type
```

Comment:
Auto-generated documentation will pick these comments up if you use triple slashes and put them before the fields they describe, e.g.

```
    /// Unique code identifying error type
    code: i32,
````